internal/pkg/cli/svc_package.go

```diff
@@ -117,6 +120,22 @@ func newPackageSvcOpts(vars packageSvcVars) (*packageSvcOpts, error) {
 	return opts, nil
 }
 
+func minifyRawManifest(ws manifestReader, workloadName string) ([]byte, error) {
+	raw, err := ws.ReadWorkloadManifest(workloadName)
+	if err != nil {
+		return nil, fmt.Errorf("read manifest file for %s: %w", workloadName, err)
+	}
+	var cleaned any
+	if err := yaml.Unmarshal(raw, &cleaned); err != nil {
+		return nil, fmt.Errorf("unmarshal to minify manifest file for %s: %w", workloadName, err)
```

Comment:
```suggestion
		return nil, fmt.Errorf("unmarshal to minimized manifest file for %s: %w", workloadName, err)
```

How does this sound?

Reply:
Deleted this function after addressing @iamhopaul123's feedback https://github.com/aws/copilot-cli/pull/3691#discussion_r907880945